libs/agno/agno/storage/session/workflow.py

```diff
@@ -59,3 +60,201 @@ def from_dict(cls, data: Mapping[str, Any]) -> Optional[WorkflowSession]:
             created_at=data.get("created_at"),
             updated_at=data.get("updated_at"),
         )
+
+
+@dataclass
+class WorkflowSessionV2:
+    """Workflow Session V2 for sequence-based workflows"""
+
+    # Session UUID - this is the workflow_session_id that gets set on agents/teams
+    session_id: str
+    # ID of the user interacting with this workflow
+    user_id: Optional[str] = None
+
+    # ID of the workflow that this session is associated with
+    workflow_id: Optional[str] = None
+    # Workflow name
+    workflow_name: Optional[str] = None
+
+    # Workflow runs - stores all WorkflowRunResponse objects
+    runs: Optional[List[Dict[str, Any]]] = None
```

Comment:
should be of type `WorkflowRunResponse`

Reply:
but this is being stored as a serialized dict in the runs column in the db.

Reply:
Yes but then we serialize it when we store it. Not on the object